Misc/NEWS.d/next/Library/2025-04-01-13-44-26.gh-issue-131938.dm4Suq.rst

```diff
@@ -0,0 +1,3 @@
+:mod:`xml.etree.ElementTree`: update the error message when an element to
+remove via :meth:`Element.remove <xml.etree.ElementTree.Element.remove>` is
+not found. Patch by Bénédikt Tran.
```

Comment:
I don't think that this change deserves a NEWS entry.

Reply:
Well, it's still a *possible* change for doctests so I don't think it hurts. A What's New is an overkill but a small NEWS is fine I think.